gradoop-common/src/main/java/org/gradoop/common/model/impl/id/GradoopIdList.java

```diff
@@ -36,24 +38,65 @@
  */
 public class GradoopIdList implements Iterable<GradoopId>, Value {
```

Comment:
I think we should rename this into `GradoopIds` and mention the set property in the doc or rename it to `GradoopIdSet`